src/include/daos/dtx.h

```diff
@@ -42,6 +42,23 @@ enum dtx_target_flags {
 	DTF_RDONLY			= (1 << 0),
 };
 
+enum dtx_grp_flags {
+	/* The group only contains read-only operations for the DTX. */
+	DGF_RDONLY			= (1 << 0),
+};
+
+enum dtx_mbs_flags {
+	/* The targets that are modified by the distributed transaction
+	 * are in the same single redundancy group.
+	 */
+	DMF_MODIFY_SRDG			= (1 << 0),
+	/* The MDS contains the leader information, used for distributed
```

Comment:
S/MDS/MBS/  ?
and just confirm for below "stand-alone modification" means same as tx for single RDG?
if DMF_CONTAIN_LEADER only set/un_set when DMF_MODIFY_SRDG == false/true, can it only use one flag?

Reply:
"stand-along-modification" means the update/punch without explicit TX. So "DMF_CONTAIN_LEADER" is not the same as "DMF_MODIFY_SRDG".